diff -up tiff-4.6.0/Makefile.am.1~ tiff-4.6.0/Makefile.am
--- tiff-4.6.0/Makefile.am.1~	2023-09-06 22:02:07.570180975 +0200
+++ tiff-4.6.0/Makefile.am	2023-09-06 22:09:58.788205462 +0200
@@ -60,7 +60,7 @@ distcheck-hook:
 	rm -rf $(distdir)/_build/cmake
 	rm -rf $(distdir)/_inst/cmake
 
-SUBDIRS = port libtiff tools build contrib test doc tools/unsupported
+SUBDIRS = port libtiff tools build test doc tools/unsupported
 
 
 NEW_LIBTIFF_RELEASE_DATE=$(shell date +"%Y%m%d")
diff -up tiff-4.6.0/Makefile.in.1~ tiff-4.6.0/Makefile.in
--- tiff-4.6.0/Makefile.in.1~	2023-09-06 22:02:07.570180975 +0200
+++ tiff-4.6.0/Makefile.in	2023-09-06 22:10:04.035261154 +0200
@@ -430,7 +430,7 @@ EXTRA_DIST = \
 	libtiff-4.pc.in
 
 @TIFF_DOCS_TRUE@dist_doc_DATA = $(docfiles)
-SUBDIRS = port libtiff tools build contrib test doc tools/unsupported
+SUBDIRS = port libtiff tools build test doc tools/unsupported
 NEW_LIBTIFF_RELEASE_DATE = $(shell date +"%Y%m%d")
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libtiff-4.pc
